### DISCUSSION TITLE
How about adding a file to show SQL components coverage?

### DISCUSSION THREAD

**Discussion by MRGRAVITY817** [gluesql/gluesql#345] [Ideas]
I was thinking it would be nice to have a visual chart or report file for showing how much the GlueSQL is covering standard SQL functionalities(expressions, functions, grammars, etc). So that other contributers can consider which part of the functionalities they should add/enhance/implement or fix.

Let me know if this is a good idea or not. And also I wanna know if there's any awesome tool to do this.

**Comment by panarch** [comment]
That is definitely a good idea.
If there exists a good judgement tool to apply, it would be awesome to use that.

e.g. 
https://html5test.com/results/desktop.html

For the case of SQL syntax, I have no idea about the tool.
